libbeat/docs/https.asciidoc

```diff
@@ -10,66 +10,152 @@
 //// This content is structured to be included as a whole file.
 //////////////////////////////////////////////////////////////////////////
 
-To secure the communication between {beatname_uc} and Elasticsearch, you can use
-HTTPS and basic authentication. Basic authentication for Elasticsearch is
-available when you enable {security} (see
-{ref}/secure-cluster.html[Secure a cluster] and <<securing-beats>>).
-If you aren't using {security}, you can use a web proxy instead.
+When sending data to a secured cluster through the `elasticsearch`
+output, {beatname_uc} can use any of the following authentication methods:
 
-Here is a sample configuration:
+* Basic authentication credentials (username and password).
+* Token-based API authentication.
+* A client certificate.
 
+Authentication is specified in the {beatname_uc} configuration file:
+
+* To use *basic authentication*, specify the `username` and `password` settings under `output.elasticsearch`.
+For example:
++
+--
 ["source","yaml",subs="attributes,callouts"]
 ----------------------------------------------------------------------
 output.elasticsearch:
-  username: {beatname_lc} <1>
-  password: verysecret <2>
-  protocol: https <3>
-  hosts: ["elasticsearch.example.com:9200"] <4>
+  hosts: ["localhost:9200"]
```

Comment:
If you remove `protocol`, might be good to show https:// in the hosts setting. I'd also use something other than localhost (though I see we do that elsewhere).